frontend/src/components/RideModal/RideModal.tsx

```diff
@@ -60,6 +68,7 @@ const RideModal = () => {
         title={['Add a Ride', 'Available Drivers', 'Add a Ride']}
```

Comment:
I just realized that this form has a working date & time selector. It is different from the one used in the mini-cal. Perhaps we should combine the two in a later PR for consistency.

The modals also have different closing buttons & margins so I'll try to update them in a different PR to be consistent with designs.

In the 2nd slide to select available drivers, there is no way to cancel selecting a driver other than clicking the back button. Perhaps add another radio button for NO DRIVER YET or something. I'm also curious about how the screen would look like as the number of drivers increase. Maybe limit the number of drivers per line and maybe it multiple lines.

The form allows the ride to start and end in the same location. This probably isn't intended and should result in a form error.

Reply:
yeah, i just realized that the minical date selector is supposed to integrate with the add a ride modal lol, ill make a PR to integrate the two after merging this one in so we won't need the date input anymore